<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputPresentation</class>
 <widget class="QDialog" name="OutputPresentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Victor Hunt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/img/img/Ticket.png</normaloff>:/img/img/Ticket.png</iconset>
  </property>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>720</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#54C475; 
border: none;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundConfetti">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>-20</y>
     <width>1341</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/img/confetti.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundTheme">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundGoldDrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1281</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/img/confetti-png.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CelebrationTitle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>891</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Astronout</family>
     <pointsize>120</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C49503;
-webkit-text-stroke: 1px red;</string>
   </property>
   <property name="text">
    <string>Sample Text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Congratulationsto">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Felix Titling</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Congratulations to</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WinnerName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>1211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Felix Titling</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C49503; </string>
   </property>
   <property name="text">
    <string>Juan Dela Cruz</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Congratulationsto_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Felix Titling</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Winner of</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PrizeName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>911</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Felix Titling</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C49503; </string>
   </property>
   <property name="text">
    <string>House &amp; Lot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRoll">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>720</y>
     <width>99</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Felix Titling</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: #C49503;
color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Hunt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteName">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>720</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Discard</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#D97C7C; 
border: none;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="themeSelection">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #ffffff;</string>
   </property>
  </widget>
  <zorder>BackgroundTheme</zorder>
  <zorder>BackgroundConfetti</zorder>
  <zorder>BackgroundGoldDrop</zorder>
  <zorder>btnEdit</zorder>
  <zorder>CelebrationTitle</zorder>
  <zorder>Congratulationsto</zorder>
  <zorder>WinnerName</zorder>
  <zorder>Congratulationsto_2</zorder>
  <zorder>PrizeName</zorder>
  <zorder>btnRoll</zorder>
  <zorder>btnDeleteName</zorder>
  <zorder>themeSelection</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
